docker-compose.yml

```diff
@@ -1,12 +1,24 @@
 version: '3'
 
 services:
+  db_volumefix:
```

Comment:
What's this? Is it temporarily added service?

Reply:
in userspace running podman it's necessary to fix  /var/lib/pgsql/data permission
it's needed only in local developer setup